hwbench/environment/mock.py

```diff
@@ -3,7 +3,7 @@
 
 
 class MockHardware(Hardware):
-    def __init__(self, flags: list[str] = None, cores: int = 0, cpu=None):
+    def __init__(self, flags: list[str] | None = None, cores: int = 0, cpu=None):
```

Comment:
This syntax needs python 3.10 or having `from __future__ import annotations` IIRC (same for all this commit)

Reply:
Just like I said :-) 

```

hwbench/utils/archive.py:29: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/utils/helpers.py:45: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/cpu_cores.py:83: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/cpu.py:55: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/pdu.py:41: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/bmc.py:135: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/bmc.py:159: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/bmc.py:187: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/bmc.py:202: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/vendor.py:14: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
hwbench/environment/vendors/vendor.py:15: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
hwbench/environment/vendors/vendor.py:70: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/pdus/raritan.py:34: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/mock.py:24: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/mock.py:40: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/mock.py:55: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/mock.py:71: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/hpe/hpe.py:42: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/hpe/hpe.py:96: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/hpe/hpe.py:135: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/hpe/hpe.py:188: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
hwbench/environment/vendors/dell/dell.py:23: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/vendors/dell/dell.py:75: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/turbostat.py:38: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/turbostat.py:39: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/environment/mock.py:6: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/bench/engine.py:43: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/bench/engine.py:73: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/engines/stressng.py:63: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/engines/stressng_stream.py:26: error: X | Y syntax for unions requires Python 3.10  [syntax]
hwbench/bench/benchmarks.py:274: error: X | Y syntax for unions requires Python 3.10  [syntax]
graph/graph.py:37: error: X | Y syntax for unions requires Python 3.10  [syntax]
```